examples/cra/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@pcd/eddsa-ticket-pcd": "^0.3.0",
-    "@pcd/passport-interface": "^0.8.0",
-    "@pcd/pcd-types": "^0.8.0",
-    "@pcd/zk-eddsa-event-ticket-pcd": "^0.2.1",
+    "@pcd/eddsa-ticket-pcd": "^0.4.0",
+    "@pcd/passport-interface": "^0.9.0",
+    "@pcd/pcd-types": "^0.9.0",
+    "@pcd/zk-eddsa-event-ticket-pcd": "^0.3.0",
```

Comment:
unfortunate that it seems like we have to do this manually

Reply:
I will figure out how to automate this. Probably when figuring out how to hook testing these builds into CI.